### ISSUE TITLE
8$: disallow 'import type a = mynamespace;'

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1139]
The following TypeScript code is invalid:
```typescript
namespace mynamespace {}
import type a = mynamespace;
```
The `type` keyword should be removed from the namespace alias.
